Sources/StreamChat/Workers/ChannelListUpdater.swift

```diff
@@ -14,10 +14,13 @@ class ChannelListUpdater<ExtraData: ExtraDataTypes>: Worker {
     ///
     func update(channelListQuery: _ChannelListQuery<ExtraData.Channel>, completion: ((Error?) -> Void)? = nil) {
         apiClient
-            .request(endpoint: .channels(query: channelListQuery)) { (result: Result<ChannelListPayload<ExtraData>, Error>) in
+            .request(endpoint: .channels(query: channelListQuery)) { [weak self] (result: Result<
+                ChannelListPayload<ExtraData>,
+                Error
+            >) in
```

Comment:
this is very weird formatting... is it the formatter's work?

Reply:
yep, it's a formatter 🙂